src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -404,6 +399,14 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     }
     resourceLoader.purgeUnusedCacheEntriesAsync(); // Don't await - it's fine to run in background
 
+    if (resourceLoader.bootConfig.icuDataMode == ICUDataMode.Sharded) {
+      MONO.mono_wasm_setenv('__BLAZOR_SHARDED_ICU',  '1');
+
+      if (resourceLoader.startOptions.applicationCulture) {
+        // If a culture is specified via start options use that to initialize the Emscripten \  .NET culture.
+        MONO.mono_wasm_setenv('LANG',  `${resourceLoader.startOptions.applicationCulture}.UTF-8`);
```

Comment:
This is interesting. Were we missing a feature before? How do you know this magic string format is correct - where does the info come from?

Reply:
This was one of the fallouts of the change. Previously this value influenced the icuData string and we expected the user to configure the .NET culture again using the same mechanism they used to determine the culture in JS. The Globalalization test app used this. It seemed slightly more preferable to constructing magic strings. 

However this trips our current detection so it seems worthwhile to initialize .NET with the right culture all the way. The string format is based on emscripten's implementation: https://github.com/emscripten-core/emscripten/blob/d9c79342262a0b05175ad1c029f4528bc8e1d136/src/library_wasi.js#L20